scripts/ci/jenkins_pipeline_csi

```diff
@@ -25,9 +33,47 @@ pipeline {
                 sh './scripts/run_yamlcheck.sh'
             }
         }
-        stage ('CSI-controller & node: Build and push images') {
+        stage ('Build docker images') {
+            steps {
+                script {
+                    sh 'bash ./scripts/ci/run_variables_verification.sh'
+                    branch = sh(script:"echo $GIT_BRANCH | sed 's|/|.|g'",returnStdout:true).trim()
+                    specific_tag = "${IMAGE_VERSION}_b${BUILD_NUMBER}_${branch}"
+                    if (latest_tag.contains(GIT_BRANCH)) {
+                        controller_image_latest = docker.build("${CSI_CONTROLLER_IMAGE}", "--build-arg VERSION=${IMAGE_VERSION} --build-arg BUILD_NUMBER=${BUILD_NUMBER} -f ${Dockerfile_controller} .")
+                        node_image_latest = docker.build("${DOCKER_REGISTRY}/${CSI_NODE_IMAGE}", "--build-arg VERSION=${IMAGE_VERSION} --build-arg BUILD_NUMBER=${BUILD_NUMBER} -f ${Dockerfile_node} .")
+                    }
+                    controller_image_with_generated_tag = docker.build("${DOCKER_REGISTRY}/${CSI_CONTROLLER_IMAGE}:$specific_tag", "--build-arg VERSION=${IMAGE_VERSION} --build-arg BUILD_NUMBER=${BUILD_NUMBER} -f ${Dockerfile_controller} .")
+                    node_image_with_generated_tag = docker.build("${DOCKER_REGISTRY}/${CSI_NODE_IMAGE}:$specific_tag", "--build-arg VERSION=${IMAGE_VERSION} --build-arg BUILD_NUMBER=${BUILD_NUMBER} -f ${Dockerfile_node} .")
+                }
+            }
+        }
+        stage ('push docker images') {
             steps {
-                sh './scripts/ci/build_push_images.sh build/reports/images_url'
+                script {
+                    registryUrl = "${registry_protocol}://${DOCKER_REGISTRY}"
+                    docker.withRegistry(registryUrl, registryCredential ) {
```

Comment:
is the space in `registryCredential )` on purpose?

Reply:
fixed it
done